### PR TITLE
DevWs for NodOn containing containing NodOn lighting Switch  and 2 more

### PR DESCRIPTION
**Pull request by AltyorFig**

### CODE REVIEW

File: devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy [hunk 1]
```diff
@@ -92,7 +92,8 @@ metadata {
 		fingerprint manufacturer: "LELLKI", model: "JZ-ZB-006", deviceJoinName: "LELLKI Switch 1" //LELLKI 6 Gang Switch 1
         
 		// NodOn
-		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0007, 0008, FC57", outClusters: "0021", manufacturer: "NodOn", model: "SIN-4-2-20", deviceJoinName: "NodOn Light 1"
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0007, 0008, 1000, FC57", outClusters: "0003,0006,0019", manufacturer: "NodOn", model: "SIN-4-2-20", deviceJoinName: "NodOn Light 1"
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0007, 0008, 1000, FC57", outClusters: "0003,0006,0019", manufacturer: "NodOn", model: "SIN-4-2-20_PRO", deviceJoinName: "NodOn Light 1"
```
Comment (dsobierajsk) [inline devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy:96]: formatting: add spaces between `outClusters` values

**Review by dsobierajsk — CHANGES_REQUESTED** [review]

File: devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy (hunk 1 above)
Comment (k-morawski) [reply devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy:96]: Dear @AltyorFig , could you please check @dsobierajsk feedback and make required changes in this pull request? Thanks!

Comment (AltyorFig) [reply devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy:96]: Hi @k-morawski  and @dsobierajsk , i commit this day the last change. 

Best regards

**Comment by KKlimczukS** [comment]
I'm closing this PR - since #77876 contains those changes already.

**Pull request closed without merging**